code/src/LeafletProvider/Configuration/DrawingTools/DrawMarkerConfig.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference path="./DrawConfig.ts" />
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace LeafletProvider.Configuration.DrawingTools {
+    export class DrawMarkerConfig extends DrawConfig {
+        public iconUrl: string;
```

Comment:
Note: In the future, we might need to add the iconSize as well.
(for the Future 🚀)